godotsteam.cpp

```diff
@@ -2480,12 +2470,12 @@ bool Steam::setHTTPRequestNetworkActivityTimeout(uint32 request_handle, uint32 t
 }
 
 // Sets the body for an HTTP Post request.
-uint8 Steam::setHTTPRequestRawPostBody(uint32 request_handle, const String &content_type, uint32 body_length) {
-	uint8 body = 0;
-	if (SteamHTTP()) {
-		SteamHTTP()->SetHTTPRequestRawPostBody(request_handle, content_type.utf8().get_data(), &body, body_length);
+bool Steam::setHTTPRequestRawPostBody(uint32 request_handle, const String &content_type, const String &body) {
+	if (SteamHTTP() == nullptr) {
+		return false;
 	}
-	return body;
+	auto body_utf8 = body.utf8();
+	return SteamHTTP()->SetHTTPRequestRawPostBody(request_handle, content_type.utf8().get_data(), body_utf8.ptrw(), body_utf8.size());
```

Comment:
Compiled does not like line 2478: `error: invalid conversion from 'char*' to 'uint8*' {aka 'unsigned char*'} [-fpermissive]`

Points to `body_utf8.ptrw()`